pkg/code-generator/codegen/templates/resource_client_test_template.go

```diff
@@ -4,15 +4,16 @@ import (
 	"text/template"
 )
 
-var ResourceClientTestTemplate = template.Must(template.New("resource_client_test").Funcs(Funcs).Parse(`// +build solokit
+var ResourceClientTestTemplate = template.Must(template.New("resource_client_test").Funcs(Funcs).Parse(`
+//go:build solokit
```

Comment:
only commenting on one of these:  is the new line entirely stylistic?

Reply:
We're actually converting `// +build solokit` into `//go:build solokit` which is a change in how go1.18 build constraints (https://pkg.go.dev/go/build) are evolved from earlier versions of go